.travis.yml

```diff
@@ -4,7 +4,7 @@ matrix:
   fast_finish: true
   include:
     - env: DM=~6.3
-      php: hhvm
+      php: hhvm-3.30
```

Comment:
This change appears to be unrelated. It's not mentioned in the commit message or the Phabricator ticket.

Reply:
It happened that current hhvm nightly does not support current version of composer (and soon enough not php). Travis was failing due to that.

This can be done in a separate PR, but think it's fine for this library to be done here since it's done in a separate commit.

Reply:
Sure, I just wanted to have the reason documented. :-)